gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineFindBugs.groovy

```diff
@@ -30,6 +30,7 @@ class BaselineFindBugs extends AbstractBaselinePlugin {
 
     private static final String DEFAULT_FINDBUGS_VERSION = '3.0.1'
     private static final String DEFAULT_EFFORT = 'max'
+    private static final String ANTIPATTERNS_ARTIFACT = 'com.palantir.antipatterns:antipatterns:1.0-beta.4'
```

Comment:
can users override the version of this? i guess you could do something like
````
configurations.findbugsPlugins.resolutionStrategy {
  force 'com.palantir.antipatterns:antipatterns:1.0'
}
````
, right? I think that's good enough.